JavaScript/chapter02/2.1 - Remove Dups/solution.js

```diff
@@ -0,0 +1,87 @@
+class node{
```

Comment:
be consistent with how you name things: you use CamelCase for `class LinkedList` below.

Reply:
Okay,thanks.